frontend/src/entities/jobs/transform/detail/transform-detail.tsx

```diff
@@ -75,7 +75,7 @@ function TransformDetail () {
     // Refresh data in the background to keep state fresh
     const isBackgroundRefreshing = useBackgroundRefresh(() => {
         dispatch(describeBatchTransformJob(name));
-    }, [dispatch], (transform.TransformJobStatus !== JobStatus.Failed && transform.TransformJobStatus !== JobStatus.Completed));
+    }, [dispatch, transform.TransformJobStatus], (transform.TransformJobStatus !== JobStatus.Failed && transform.TransformJobStatus !== JobStatus.Completed));
```

Comment:
why do we have to pass in the status here? wont it re-render anyways when the condition changes?

Reply:
I was seeing an issue with training jobs where it wasn't triggering an updating and a clearing of the interval. I can run some additional tests though.